packages/tailwind/src/tailwind-presets.ts

```diff
@@ -18,29 +20,29 @@ export const kaizenTailwindTheme: KaizenTailwindTheme = {
     ...defaultTheme.dataViz,
   },
   spacing: {
-    ...defaultTheme.spacing,
-    ...twDefaultTheme.spacing,
+    ...kzSpacing,
   },
   boxShadow: {
     none: "none",
     sm: defaultTheme.shadow.small.boxShadow,
     lg: defaultTheme.shadow.large.boxShadow,
   },
   borderRadius: {
-    none: "0px",
     default: "7px",
+    none: "0px",
     "focus-ring": "10px",
     full: "100%",
   },
+
   borderWidth: {
-    "width-none": "0px",
-    "width-default": "2px",
-    "width-focus-ring": "2px",
+    "w-none": "0px",
+    "w-default": "2px",
+    "w-focus-ring": "2px",
   },
   // how do we want to handle our focus rings?
   // tailwind used a `ring` property
   borderColor: {
-    "color-solid": `${defaultTheme.border.solid.borderColor}`,
+    DEFAULT: `${defaultTheme.border.solid.borderColor}`,
```

Comment:
Curious about this. Does this mean the token is just `border-color` (or whatever it is) without a suffix?

Almost seems strange to even have a default solid border colour but that's a whole different conversation I think.

Reply:
actually `border-color:` translate to `border${your-color-prefix}` in tw so it would just be `border`. We could leave it as 'color-default` and provide DEFAULT as the base one too 🤷 or just not use DEFAULT.

Reply:
So you're telling me that `border` is only `border-color`, so if we don't provide `DEFAULT` then `border` gives no styles? 😂 
I just wonder if it even makes sense to have a default border colour at all. Is that a pattern/guideline we have? That if we design anything new with a border, it will have this colour unless you choose to override it?

Reply:
The border utility can take a color, size and style as a prefix

Reply:
lul, just found out that as DEFAULT border does not do what we wanted, also we've both decided we should just flat out avoid defaults